code/game/objects/effects/contraband.dm

```diff
@@ -662,4 +665,21 @@
 	desc = "A Syndicate-commissioned poster that uses Syndie Moth(TM?) to tell the viewer to keep the nuclear authentication disk unsecured. No, we aren't doing that. It's signed by 'AspEv'."
 	icon_state = "poster49"
 
+/obj/structure/sign/poster/official/festive
+	name = "Celebrate the Day!"
+	desc = "A poster informing the reader every day is a gift, and not to waste it."
+	icon = 'icons/obj/festive_posters.dmi'
+	icon_state = "generic"
+
+/obj/structure/sign/poster/official/festive/Initialize()
+	if(!length(SSevents.holidays))
+		return
+	var/cele_holiday = pick(SSevents.holidays)
```

Comment:
You don't need to make this var

Reply:
i feel like the line would be way too long if i left the pick function in line